force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,12 +256,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
-        // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
-
-        //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        for(string str : inputList){
+            stringSet.add(str);
+        }
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
Good work. Take a look at the Set class constructor.  You can solve this with one line of code and not lose any readability. In my opinion.